test/framework/prometheus.go

```diff
@@ -494,9 +538,18 @@ func (f *Framework) WaitForPrometheusFiringAlert(ns, svcName, alertName string)
 	return nil
 }
 
+type targetHealth string
+
+const (
+	healthGood targetHealth = "up"
```

Comment:
we shouldn't rename these, the variable names should just be `healthUp` and `healthDown`

Reply:
I mirrored what is exposed by Prometheus scrape package here: https://github.com/prometheus/prometheus/blob/master/scrape/target.go#L43-L44

Reply:
ok makes sense, at least consistent in one way :)